runtime/decimal.c

```diff
@@ -57,6 +57,20 @@ TaggedPtrPanicCode _bal_decimal_div(TaggedPtr tp1, TaggedPtr tp2) {
     return finish(&d, &cx); 
 }
 
+TaggedPtrPanicCode _bal_decimal_rem(TaggedPtr tp1, TaggedPtr tp2) {
+    decQuad d;
+    decContext cx;
+    initContext(&cx);
+    decQuadRemainder(&d, taggedToDecQuad(tp1), taggedToDecQuad(tp2), &cx);
+    return finish(&d, &cx);    
+}
+
+TaggedPtrPanicCode _bal_decimal_neg(TaggedPtr tp) {
```

Comment:
When can this panic? I don't think it can.

Reply:
Yes, this cannot panic, But to handle -Zero case and be consistent with other cases  I passed result and cx(which is NULL) to finish method.

I changed it as per the below comment.